mono/utils/mono-time.c

```diff
@@ -142,7 +142,16 @@ gint64
 mono_100ns_ticks (void)
 {
 	struct timeval tv;
-#ifdef CLOCK_MONOTONIC
+#if defined(PLATFORM_MACOSX)
+	/* http://developer.apple.com/library/mac/#qa/qa1398/_index.html */
+	static mach_timebase_info_data_t timebase;
+	guint64 now = mach_absolute_time ();
```

Comment:
Out of curiosity, does it matter for this function that `mach_absolute_time ()` stops updating when the device goes to sleep?